### DISCUSSION TITLE
Deprecated DynamicLinks and Auth

### DISCUSSION THREAD

**Discussion by sergiocampama** [firebase/firebase-ios-sdk#12726] [General]
It is our understanding that Auth sends email verification links and password resets using DynamicLinks. What will happen to those when DynamicLinks is removed later this year? Is there a plan for that?

**Comment by paulb777** [comment]
See https://github.com/firebase/firebase-ios-sdk/issues/12556#issuecomment-2005211499

**Reply by sergiocampama** [reply]
thanks! is there a related issue tracking this, given the one you linked is closed?

**Reply by paulb777** [reply]
I'll convert this discussion item to an issue for tracking
